### ISSUE TITLE
Release version in maven

### ISSUE THREAD

**Issue by clochardpagan** [kefirfromperm/grails-asynchronous-mail#95]
Last two commits were applied about two years ago. One of which fixes error in cluster setup. We can build plugin by hand and inject it to current systems, but it will be much more convenient if maven artifact will be available in repo.

**Comment by xpusostomos** [comment]
Yes I'm wondering if this project is dead. I might have to fork this project if there's no responses from the author.
